pkg/volume/csi/csi_client.go

```diff
@@ -162,8 +162,11 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 			return true, nil
 		}
 		// kubelet plugin registration service not implemented is a terminal error, no need to retry
-		if strings.Contains(getNodeInfoError.Error(), "no handler registered for plugin type") {
-			return false, getNodeInfoError
+		if getNodeInfoError != nil {
+			if strings.Contains(getNodeInfoError.Error(), "no handler registered for plugin type") {
+				return false, getNodeInfoError
+			}
+			klog.V(4).Info(getNodeInfoError.Error())
 		}
 		// Continue with exponential backoff
```

Comment:
nit: Worth logging something here?

Reply:
Please take a look at the updated PR.
If there is no error, probably we don't need extra logging.